lib/node_modules/@stdlib/math/base/special/expm1/README.md

```diff
@@ -75,6 +75,96 @@ for ( i = 0; i < 100; i++ ) {
 
 <!-- /.examples -->
 
+<!-- C interface documentation. -->
+
+* * *
+
+<section class="c">
+
+## C APIs
+
+<!-- Section to include introductory text. Make sure to keep an empty line after the intro `section` element and another before the `/section` close. -->
+
+<section class="intro">
+
+</section>
+
+<!-- /.intro -->
+
+<!-- C usage documentation. -->
+
+<section class="usage">
+
+### Usage
+
+```c
+#include "stdlib/math/base/special/expm1.h"
+```
+
+#### stdlib_base_expm1( x )
+
+Computes `exp(x) - 1`, where `exp(x)` is the natural [exponential function][exponential-function]
+
+```c
+double out = stdlib_base_expm1( 0.2 );
+// returns ~0.221
+
+out = stdlib_base_expm1( -9.0 );
+// returns ~-1.0
+```
+
+The function accepts the following arguments:
+
+-   **x**: `[in] double` input value.
+
+```c
+double stdlib_base_expm1( const double x );
+```
+
+</section>
+
+<!-- /.usage -->
+
+<!-- C API usage notes. Make sure to keep an empty line after the `section` element and another before the `/section` close. -->
+
+<section class="notes">
+
+</section>
+
+<!-- /.notes -->
+
+<!-- C API usage examples. -->
+
+<section class="examples">
+
+### Examples
+
+```c
+#include "stdlib/math/base/special/expm1.h"
+#include <stdlib.h>
+#include <stdio.h>
+
+int main() {
+    double x;
+    double v;
+    int i;
+    
+    for ( i = 0; i < 100; i++ ) {
+        x = (double)rand() / (double)RAND_MAX * 10.0 - 5.0;
```

Comment:
```suggestion
        x = ( ( (double)rand() / (double)RAND_MAX ) * 10.0 ) - 5.0;
```

We're fairly liberal with our use of parentheses in order to make clear the order of operations.